javascript/index.js

```diff
@@ -1,3 +1,36 @@
+const navbookList = document.querySelector('.listBook');
+const navAddBook = document.querySelector('.addBook');
+const navContact = document.querySelector('.contactNav');
+console.log(navContact);
+console.log(navAddBook);
+console.log(navbookList);
+// const links = document.querySelectorAll('.navbar-list');
+// const containers = document.querySelectorAll('.container');
+// function hideAllContainers() {
+//   containers.forEach((container) => {
+//     container.style.display = 'none';
+//   });
+// }
+
+// links.forEach((link) => {
+//   link.addEventListener('click', (e) => {
+//     const linkClass = e.target.classList[1];
+//     console.log(linkClass);
+//     hideAllContainers();
+//     containers.forEach((container) => {
+//       if (linkClass === container.id) {
+//         container.style.display = 'none';
+//       } else {
+//         container.style.display = 'block';
+//       }
+//     });
+//   });
+// });
+// window.addEventListener('DOMContentLoaded', () => {
+//   hideAllContainers();
+//   containers[0].style.display = 'block';
+// });
```

Comment:
- [ ]  please always do well to remove commented code from your code base so as to not make your code messy and unprofessional 😉

Reply:
This change has been done well